lisp/tree-sitter-query.el

```diff
@@ -0,0 +1,119 @@
+;; tree-sitter-query.el --- tools for running queries live -*- lexical-binding: t; coding: utf-8 -*-
+
+;; Copyright (C) 2019  Tuấn-Anh Nguyễn
+;;
+;; Author: Jorge Javier Araya Navarro <jorgejavieran@yahoo.com.mx>
+
+;; This file contains other debug utilities for building queries and see
+;; results in a target buffer
+
+;; Code:
+
+(require 'scheme)
```

Comment:
Why scheme?

Reply:
That is because tree-sitter queries use a syntax similar to that of scheme, AFAIK, so I thought on borrowing stuff from scheme-mode